tests/tslint/dist/noBarrelImportsRule.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
```

Comment:
Still need to remove this file.

Reply:
yap, I got this from the last merge with the master branch, but will address it on my next changes if there any